projects/angular/src/forms/combobox/_combobox.clarity.scss

```
@@ -190,129 +101,55 @@
-        variables.$clr-p7-font-size,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        line-height,
-        clr-combobox-pill-line-height,
-        variables.$clr-p7-font-size + variables.$clr_baselineRem_1px,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        letter-spacing,
-        clr-combobox-pill-letter-spacing,
-        variables.$clr-p7-letter-spacing,
-        variables.$clr-use-custom-properties
-      );
-
-      @include mixins.css-var(
-        height,
-        clr-combobox-pill-height,
-        variables.$clr_baselineRem_0_875,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        margin,
-        clr-combobox-pill-margin,
-        variables.$clr_baselineRem_0_125 variables.$clr_baselineRem_0_25 0 0,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        background-color,
-        clr-combobox-pill-background-color,
-        combobox-variables.$clr-combobox-pill-background-color,
-        variables.$clr-use-custom-properties
-      );
-      border-width: variables.$clr_baselineRem_1px;
+      @include mixins.generate-typography-token(
+        'CAPTION-LG-11',
+        (
+          font-weight: combobox-variables.$clr-combobox-pill-font-weight,
+          font-size: combobox-variables.$clr-combobox-pill-font-size,
+          line-height: combobox-variables.$clr-combobox-pill-line-height,
+          letter-spacing: combobox-variables.$clr-combobox-pill-letter-spacing,
+        )
+      );
+      height: combobox-variables.$clr-combobox-pill-height;
+      margin: combobox-variables.$clr-combobox-pill-margin;
+      background-color: combobox-variables.$clr-combobox-pill-background-color;
+      border-width: tokens.$cds-global-space-1;
       border-style: solid;
+      align-items: center;
 
-      // ATM Remove fallback and property
-      @include mixins.css-var(
-        align-items,
-        clr-combobox-pill-align-items,
-        baseline,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        border-radius,
-        clr-combobox-pill-border-radius,
-        combobox-variables.$clr-combobox-pill-border-radius,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        border-color,
-        clr-combobox-pill-border-color,
-        combobox-variables.$clr-combobox-pill-border-color,
-        variables.$clr-use-custom-properties
-      );
+      border-radius: combobox-variables.$clr-combobox-pill-border-radius;
+      border-color: combobox-variables.$clr-combobox-pill-border-color;
 
-      @include mixins.css-var(
-        padding,
-        clr-combobox-pill-padding,
-        0 variables.$clr_baselineRem_2px 0 variables.$clr_baselineRem_4px,
-        variables.$clr-use-custom-properties
-      );
+      padding: combobox-variables.$clr-combobox-pill-padding;
```

`--clr-combobox-pill-padding` is introduced in TM, it's safe to delete it.

```suggestion
      padding: 0 #{tokens.$cds-global-space-3};
```

---

It was intentionally introduced as part of an API improvement effort.
As such, it is not just a v16 temporary property.
I prefer to keep it.